PWGHF/vertexingHF/AliAnalysisTaskSEDvsEventShapes.h

```diff
@@ -265,8 +435,9 @@ class AliAnalysisTaskSEDvsEventShapes : public AliAnalysisTaskSE
     Int_t ffiltbit1;
     Int_t ffiltbit2;
     Double_t fphiStepSizeDeg;
+    Int_t fYearNumber; ///year number of the data taking
     
-    ClassDef(AliAnalysisTaskSEDvsEventShapes,14); // D vs. mult task
+    ClassDef(AliAnalysisTaskSEDvsEventShapes,20); // D vs. mult task
```

Comment:
this should be fixed to ClassDef(AliAnalysisTaskSEDvsEventShapes,15);
instead of 20. Could you update the PR? thanks!

Reply:
I have updated for the comment